src/assignments/launch-checklist.rst

```diff
@@ -0,0 +1,78 @@
+Assignment #5: Launch Checklist Form
+====================================
+
+Using our knowledge of forms, the DOM, and HTTP, our favorite space shuttle program has asked us to create a quick launch checklist.
+We have four fields that need to be filled out with vital information: the pilot's name, the co-pilot's name, the fuel levels, and the weight of the cargo.
+
+Our pilot, Chris, and their co-pilot, Blake, have been hard at work securing the cargo and filling the shuttle tank. All we need to do is use validation to insure that we have all of the info for the space shuttle program and make sure no one prematurely launches the shuttle.
+
+Requirements
+------------
+
+Create a Launch Checklist Form for astronauts to fill out in preparation for launch.
+The form should do the following:
+
+1. Validate the user responses with ``preventDefault()`` to insure the following:
```

Comment:
That is why I referred them back to the validation section first. That section shows them where to put it.

Reply:
Yes, but the JavaScript in the book example is placed within the HTML tags instead of in a separate ``script.js`` file. Is this how you want the studio to work, or did you want the students to code ``preventDefault()`` in ``script.js``?

Reply:
They can code ``preventDefault()`` in ``script.js``. I will add more explicit directions to add all JS into ``script.js``.